vatomcli-plugin-template/src/index.js

```diff
@@ -1,3 +1,8 @@
+import { BasePlugin } from 'vatom-spaces-plugins'
+import { BaseComponent } from 'vatom-spaces-plugins'
```

Comment:
You could reduce this to
```suggestion
import { BasePlugin, BaseComponent } from 'vatom-spaces-plugins'
```

Reply:
Ya you're right. Shortened to one line in last push